src/app/components/header.tsx

```diff
@@ -0,0 +1,5 @@
+export default function Header() {
+  return (
+    <h1 className="fixed w-full p-4 bg-teal text-4xl text-white">Murojaah</h1>
```

Comment:
let's use semantic html tags
```suggestion
    <header className="fixed w-full p-4 bg-teal text-4xl text-white">Murojaah</h1>
```

Reply:
already updated on https://github.com/RR-Ardhika/murojaah/pull/3/commits/03e94125dce8075c8df7a729973197ff0498dae9